vcd/resource_vcd_network_isolated_v2.go

```diff
@@ -0,0 +1,289 @@
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"vdc": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The name of VDC to use, optional if defined at provider level",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Network name",
+			},
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Network description",
+			},
+			"is_shared": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "NSX-V only - share this network with other VDCs in this organization. Default - false",
+			},
+			"gateway": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Gateway IP address",
+			},
+			"prefix_length": &schema.Schema{
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: "Network prefix",
+			},
+			"dns1": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "DNS server 1",
+			},
+			"dns2": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "DNS server 1",
+			},
+			"dns_suffix": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "DNS suffix",
+			},
+			"static_ip_pool": &schema.Schema{
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Description: "IP ranges used for static pool allocation in the network",
+				Elem:        networkV2IpRange,
+			},
+		},
+	}
+}
+
+// resourceVcdNetworkIsolatedV2Create
+func resourceVcdNetworkIsolatedV2Create(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	vcdClient := meta.(*VCDClient)
+
+	_, vdc, err := vcdClient.GetOrgAndVdcFromResource(d)
+	if err != nil {
+		return diag.Errorf("[isolated network create v2] error retrieving VDC: %s", err)
+	}
+
+	networkType, err := getOpenApiOrgVdcIsolatedNetworkType(d, vdc)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	orgNetwork, err := vdc.CreateOpenApiOrgVdcNetwork(networkType)
+	if err != nil {
+		return diag.Errorf("[isolated network create v2] error creating Org Vdc isolated network: %s", err)
+	}
+
+	d.SetId(orgNetwork.OpenApiOrgVdcNetwork.ID)
+
+	return resourceVcdNetworkIsolatedV2Read(ctx, d, meta)
+}
+
+// resourceVcdNetworkIsolatedV2Update
+func resourceVcdNetworkIsolatedV2Update(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	vcdClient := meta.(*VCDClient)
+
+	_, vdc, err := vcdClient.GetOrgAndVdcFromResource(d)
+	if err != nil {
+		return diag.Errorf("[isolated network update v2] error retrieving VDC: %s", err)
```

Comment:
Maybe more correct to use this order in these messages:
```suggestion
		return diag.Errorf("[isolated network v2 - update] error retrieving VDC: %s", err)
```

Reply:
Changed for all occurrences.